include/clang/Interpreter/CppInterOp.h

```diff
@@ -535,9 +550,17 @@ namespace Cpp {
       : m_Type(type), m_IntegralValue(integral_value) {}
   };
   /// Builds a template instantiation for a given templated declaration.
-  CPPINTEROP_API TCppScope_t InstantiateTemplate(TCppScope_t tmpl,
-                                                 TemplateArgInfo* template_args,
-                                                 size_t template_args_size);
+  /// Offers a single interface for instantiation of class, function and
+  /// variable templates
+  ///\param[in] tmpl - Uninstantiated template class/function
+  ///\param[in] template_args - Pointer to vector of template arguments stored
+  /// in the \c TemplateArgInfo struct \param[in] template_args_size - Size of
+  /// the vector of template arguments passed as \c template_args
```

Comment:
We seem to be missing a new line somewhere here and indent the continued paragraph describing `template_args`.

Reply:
done